Src/LinearSolvers/MLMG/AMReX_MLNodeLaplacian.cpp

```diff
@@ -1357,6 +1400,11 @@ MLNodeLaplacian::Fsmooth (int amrlev, int mglev, MultiFab& sol, const MultiFab&
 
     const iMultiFab& dmsk = *m_dirichlet_mask[amrlev][mglev];
 
+    bool regular_coarsening = true;
+    if (amrlev == 0 and mglev > 0) {
+        regular_coarsening = mg_coarsen_ratio_vec[mglev-1] == mg_coarsen_ratio;
+    }
+
```

Comment:
This is used in a lot of places now.  Maybe we can put it in function say `bool is_regular_coarsening(amrlev, fine_mglev)`.